vtex/admin-ui
# Apply design updates on Shell components

#### What is the purpose of this pull request?


**List of changes applied:**
- Sidebar expanding behavior on hovering a section when the sidebar is collapsed.
- Change selected section style to have a blue border indicating it
- Add an arrow to indicate which section is with the Sidebar second level opened
- Change `fg.action.neutral` and `fg.action.neutralHover` values
- Sidebar skeleton styles when on loading state
- Align the Topbar icon with the sidebar section icons.
- Topbar and Sidebar Skeleton changes
- Spacing in general to a pixel perfect shell

#### How should this be manually tested?


Check the storybook preview and go to the Sidebar story. If you want to compare with the latest version [you can check this workspace](https://shellupdates.surge.sh/?path=/story/admin-ui-sidebar--playground)

> 💡 Tip: You can press `a`, `s`, and `t` to hide Storybook widgets and to show only the story content. It's better to simulate the real scenario.

#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
